examples/test_data_gen/test_data_gen_local/config.ini

```diff
@@ -0,0 +1,11 @@
+; should_skip_doc_split = True
+documents_folder = "D:\proj\github\ms\promptflow\docs\how-to-guides"
+document_chunk_size = 1024
+; cspell: ignore luyao
+document_nodes_output_path = "D:\proj\github\ms\promptflow\examples\test_data_gen\document-nodes-test.jsonl"
```

Comment:
line 5 and line 11 can share the same output folder, no need to ask user to specify jsonl file name.